src/firefly/js/ui/tap/AdvancedADQL.jsx

```diff
@@ -287,24 +286,22 @@ export function AdvancedADQL({adqlKey, defAdqlKey, serviceUrl, capabilities, sty
                                     </Stack>
                                 </Tooltip>
                                 <Stack spacing={1}>
-                                        <Tooltip placement='right'
-                                                 title={
-                                                     <Stack>
-                                                         <Typography>
-                                                             When selecting a column from the Schema browser use the full qualified name
-                                                         </Typography>
-                                                         <Typography color='warning'>
-                                                             Fully-qualified column names are recommended for table joins
-                                                         </Typography>
-                                                     </Stack>
-                                                 }>
-                                            <Switch {...{ size:'md', sx:{alignSelf:'flex-start'},
-                                                endDecorator: fullyQualified? 'Fully qualified column name' : 'Column name only', checked:fullyQualified,
-                                                onChange: (ev) => {
-                                                    setFullQualified(!getFullQualified());
-                                                },
-                                            }} />
-                                        </Tooltip>
+                                    <Switch {...{ size:'md', sx:{alignSelf:'flex-start'},
+                                        endDecorator: (
+                                            <Stack>
+                                                <Typography>
+                                                    Insert fully-qualified column names (recommended for table joins)
+                                                </Typography>
+                                                <FormHelperText>
+                                                    When selecting a column from the Schema browser use the full qualified name
```

Comment:
I like the use of FormHelperText but we should use it consistently throughout.  I don't particularly like how it looks as is.  But, we can always apply a change in theme to have it effect everywhere in the app.